build.fsx

```diff
@@ -72,21 +78,16 @@ Target.create "InstallClient" (fun _ ->
     runDotNet "restore" clientPath
 )
 
-Target.create "RestoreServer" (fun _ ->
```

Comment:
.NET Core [implicitly runs restore](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build#implicit-restore) prior to build